### ISSUE TITLE
[ui5-toolbar][A11y]: toolbar keyboard interaction does not align with WCAG requirement

### ISSUE THREAD

**Issue by caroline-ferri** [UI5/webcomponents#12945]
### Bug Description

The ui5-toolbar applies role='toolbar' to the web component. However, the toolbar items are not navigable via arrow keys. This is violates the expectations documented in [WCAG Keyboard Interaction](https://www.w3.org/WAI/ARIA/apg/patterns/toolbar/#keyboardinteraction) and [Mozilla Accessibility Toolbar Role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/toolbar_role#description). 

According to the following resources, a control that uses role="toolbar" should navigate between the inner controls via "Left Arrow" / "Right Arrow" keys if horizontal and "Up Arrow" / "Down Arrow" keys if vertical. Currently, if a toolbar has multiple toolbar items, the user can only navigate to the next item via the "Tab" key. 

To align with WCAG requirements, we expect the toolbar items to be navigable via arrow keys. This issue is reproducible in the current [Toolbar sample](https://ui5.github.io/webcomponents/components/Toolbar/#always-overflowing-items).

### Affected Component

ui5-toolbar

### Expected Behaviour

Toolbar items should be navigable via arrow keys

### Isolated Example

https://js-kh1uvjsa.stackblitz.io

### Steps to Reproduce

1. navigate to stackblitz
2. tab to Button 1
3. Try to navigate to Button 2, Button  and Button 4 via the arrow keys
...

### Priority

Medium

### UI5 Web Components Version

2.18.0

### Browser

Chrome

### Operating System

macOS

### Additional Context

This is a customer reported a11y violation. 

### Organization

SuccessFactors

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by NHristov-sap** [comment]
Hi @UI5/ui5-team-ctr-pir ,

Just to follow the process - I saw that there is pending change related to this issue, but please take a look of it after that.

Best Regards,
Nikolay Hristov

**Comment by caroline-ferri** [comment]
@plamenivanov91 , do you have any updates or an ETA for this fix? This is a customer reported a11y violation so we would like to provide a fix timeline when possible.

**Comment by plamenivanov91** [comment]
Hello @caroline-ferri ,

We are working on the issue in the PR.

Internal JIRA ID BGSOFUIPIRIN-7018

Regards,
Plamen Ivanov

**Comment by caroline-ferri** [comment]
@plamenivanov91 I see the internal JIRA BGSOFUIPIRIN-7018 is still not completed. Do you know when we can expect this fix by? I have an open customer incident that requires additional information about a fix version/timeline.

**Comment by caroline-ferri** [comment]
@plamenivanov91 , following up on this. Do you have an ETA for this fix? I want to communicate an ETA to the customer.

**Comment by caroline-ferri** [comment]
@plamenivanov91 , do you know what version this fix will be released in? I see the PR has been opened for some time now. Please provide an update when you can.

**Comment by judith9209** [comment]
@plamenivanov91  We have received a customer ticket. We will require the code fix by end of July. Would you confirm if this is feasible?

**Comment by dkris-sap** [comment]
@plamenivanov91 , We have customer incident open since January of this year and is way past the SLA timelines. Could you please merge this and provide us a version where it'll be available for us?
